content/en/database_monitoring/recommendations/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Recommendations
+description: View recommendations to proactively address issues in your system
+
+
+---
```

Comment:
To confirm, this feature is in Preview, right? If yes, we can add a banner that states it's in Preview.

Reply:
We're sending out a GA announcement, so I'll actually remove the Preview banner from the UI instead.